modules/kuberay-cluster/variables.tf

```diff
@@ -29,6 +29,12 @@ variable "db_region" {
   default     = "us-central1"
 }
 
+variable "cloudsql_instance_name" {
```

Comment:
I guess we should also make here to be the var and pass it from application layer: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/modules/cloudsql/main.tf#L64C16-L64C35

Reply:
already exists: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/modules/cloudsql/variables.tf#L26